Sources/Rendering/Core/VolumeMapper/index.js

```diff
@@ -147,6 +147,8 @@ const DEFAULT_VALUES = {
   imageSampleDistance: 1.0,
   maximumSamplesPerRay: 1000,
   autoAdjustSampleDistances: true,
+  initialInteractionScale: 16.0,
```

Comment:
I can be anything but I have chosen 16 because I think that users usually want a smooth first interaction and a high quailty rendering when not interacting
This is only the scale at first interaction, it has not a big impact and the scale will decrease or increase depending on the framerate and eventually be set to 1 if possible

Reply:
should probably be 1 to be backward compatible